### PR TITLE
🐛 Fixed GScan performance

### LINKED ISSUE #159: 050-koenig-css-classes.js checker reads all asset files from disk
This can slow down Ghost, because it reads any asset from disk.

e.g. videos, images etc

Bug was introduced in the new checker

> 050-koenig-css-classes.js

The checker reads any file from disk 🤨It only has to read css/hbs files.

### PR DESCRIPTION
**Pull request by kirrg001**
closes #159

- the checker read any asset from disk
- added restriction for .hbs and .css files

### CODE REVIEW

File: lib/checks/050-koenig-css-classes.js [hunk 1]
```diff
@@ -20,8 +20,12 @@ checkKoenigCssClasses = function checkKoenigCssClasses(theme, options, themePath
 
     _.each(ruleSet, function (check, ruleCode) {
         if (theme.files) {
-            // Cheeck CSS files for presence of the classes
+            // Check CSS files and HBS files for presence of the classes
             _.each(theme.files, function (themeFile) {
+                if (['.css', '.hbs'].indexOf(themeFile.ext) === -1) {
```
Comment (allouis) [inline lib/checks/050-koenig-css-classes.js]: would `includes` be more readable here? :shrimp:

Comment (gargol) [reply lib/checks/050-koenig-css-classes.js]: had exact same comment yesterday :smile:  only noticed today that you have to 'submit review' manually on Github for the comment to go out :confused:

[the remark this answers is not in the document]
Comment (kirrg001) [reply lib/checks/050-koenig-css-classes.js]: If you both want it, i'll replace it 🤠

**Comment by kirrg001** [comment]
@gargol Updated. Wanne test & squash? 👻

**Comment by gargol** [comment]
@kirrg001 cool. sorry for being a bit picky :smile: thinking if we go into 'includes' usage, maybe lets update other `=== -1` usages in that module e.g.: https://github.com/TryGhost/gscan/pull/160/files#diff-63d45ee8f248ba7225b2b97de1c16d3bR33 and  https://github.com/TryGhost/gscan/pull/160/files#diff-63d45ee8f248ba7225b2b97de1c16d3bR42 :thinking: 
testing it at the moment will come back with results in a minute :wink:

**Comment by kirrg001** [comment]
> cool. sorry for being a bit picky 😄

no problem 🤠 We can stick with `includes` from now on. We can update any `indexOf` occurrences if we see them while developing.

**Comment by gargol** [comment]
The fix works great. Tested it by copying my 'Downloads' folder into the theme. Results for `master`:
```
➜  gscan git:(master) time node bin/cli.js ../Ghost/content/themes/casper
Checking directory...
...
✓ 110 Passed Rules
node bin/cli.js ../Ghost/content/themes/casper  83,59s user 6,16s system 99% cpu 1:29,97 total
```
`83,59s` :scream: 

and with the fix:
```
➜  gscan git:(fix-checker-koenig) time node bin/cli.js ../Ghost/content/themes/casper
Checking directory...
...
✓ 110 Passed Rules
node bin/cli.js ../Ghost/content/themes/casper  0,55s user 0,05s system 119% cpu 0,510 total
```
`0,55s` :raised_hands:

**Pull request merged**